src/ConferenceList.jsx

```diff
@@ -0,0 +1,17 @@
+import React from "react";
+import ConferenceItem from "./ConferenceItem";
+import conferences from "./conferenceDataMock.json";
+
+const ConferenceList = () => {
+  return (
+    <div className="conference list">
+      <ul>
+        {conferences.map(conference => (
```

Comment:
It is good practice to try and split your data handling/storing and the presentation of the data.

In this case it's a good idea to pass your conferences array from `App` as a prop  to your `ConferenceList` component. This will make the component more reusable.

Reply:
so pass import in `App`, pass down to `ConferenceList` and then from there to `ConferenceItem`?

Reply:
I don't think we need to reuse the code so for now we'll leave it as is so its easier to read

Reply:
@MerrettM @sophiambolah  Yes, right now the data is being imported from a static document, but if this ends up being an api call you will have to worry about where that logic lives and when you request it. It is good habit to always separate concerns, however if you feel like you it's easier to read like this for the time being go with it.

Reply:
@robojeda sorry I may have confused them a bit when I went to speak to them earlier and I mentioned too much about reusability and not much about the presentation/data handling. My fault! I agree, once we bring in the api calls and have more logic to perform it will be a lot neater to have it in a separate place, so it's a good idea to do it now!